src/pages/patientView/mutation/VAFLineChartUtils.ts

```diff
@@ -20,16 +21,43 @@ function isPointBasedOnRealVAF(d: { mutationStatus: MutationStatus }) {
     );
 }
 
+function splitMutationsBySampleGroup(
```

Comment:
can you explain what this method does? I want to make a suggestion on the naming but I want to make sure I understand first what its purpose is

Reply:
For example, when user selects "Sample type" from the Group By dropdown, then a mutation that contains all samples (P04_Pri, P04_Rec1, P04_Rec2, P04_Rec3) will be split into 2 different mutations (P04_Pri) and (P04_Rec1, P04_Rec2, P04_Rec3) as the samples are part of 2 different groups (P04_Pri is Primary while the rest are Recurrent). This is needed because the purpose of Group by is to draw lines only between the samples from same group.